docs/content/docs/connectors/table/formats/protobuf.md

```diff
@@ -149,9 +149,12 @@ Format Options
       <td style="word-wrap: break-word;">false</td>
       <td>Boolean</td>
       <td>
-          This option only works if the generated class's version is proto2. If this value is set to true, the format will read empty values as the default values defined in the proto file.
+          If this value is set to true, the format will read empty values as the default values defined in the proto file.
           If the value is set to false, the format will generate null values if the data element does not exist in the binary protobuf message.
-          If the proto syntax is proto3, this value will forcibly be set to true, because proto3's standard is to use default values.
+          If the proto syntax is proto3, this value needs to be set to true by users when using proto version older than 3.15 because proto3 
```

Comment:
Ack, Updated the content to align with `PbFormatOptions`

Reply:
@libenchao In case you haven't noticed, I'd also like to point out that this is a breaking change as older SQL queries without proper null checking can fail after this change. With that being said, this is a step in the right direction by making the default value to be the better performance choise

Reply:
@sharath1709 Thanks for the reminder, this is indeed a breaking change, we can add a release note to talk about it.